### ISSUE TITLE
When the page is refreshed, an error will be reported when playing automatically  

### ISSUE THREAD

**Issue by minbaowang** [wavesurfer-js/wavesurfer.js#2089]
##When the page is refreshed, an error will be reported when playing automatically 
##The scene is, in a table, click a line will have a transition, click open can automatically play. But when I refresh the page and the transition keeps auto pop-up, auto play fails
```js
     Uncaught (in promise) DOMException

**Comment by thijstriemstra** [comment]
@minbaowang what browser, operating system, version of wavesurfer?

**Comment by minbaowang** [comment]
> @minbaowang what browser, operating system, version of wavesurfer?
chrome browser  , win10 system, Latest version

**Comment by minbaowang** [comment]
> @minbaowang what browser, operating system, version of wavesurfer?
![企业微信截图_16023214356219](https://user-images.githubusercontent.com/40685955/95651380-9747ff80-0b1c-11eb-91c6-30ddb3f09c81.png)

##When I click the refresh button, the page will refresh. The transfer of Vue will be opened automatically. If the wavesurfer is set to play automatically, it will not succeed and an error will be reported；
##The code for setting is as follows：
```js
this.wavesurferObj.on("ready", function () {
	that.audioObj.duration = that.formatSeconds(that.wavesurferObj.getDuration());
	that.wavesurferObj.on("seek", function () {
	       that.audioObj.currentTime = that.formatSeconds(that.wavesurferObj.getCurrentTime());
	});
	that.wavesurferObj.on("audioprocess", function () {
	       that.audioObj.currentTime = that.formatSeconds(that.wavesurferObj.getCurrentTime());
	});
	that.$nextTick(() => {
		that.wavesurferObj.play()//自动播放
	})
})

**Comment by thijstriemstra** [comment]
> The transfer of Vue will be opened automatically. 

Please make an example that reproduces this problem that doesn't use vue.js or any other framework.

**Comment by marizuccara** [comment]
>  If the wavesurfer is set to play automatically, it will not succeed and an error will be reported；

Maybe is due to the autoplay policy?

**Comment by sundayz** [comment]
Most browsers don’t allow audio to be played automatically, they require user interaction first.

**Comment by minbaowang** [comment]
> 大多数浏览器不允许自动播放音频，它们首先需要用户交互。



> Most browsers don’t allow audio to be played automatically, they require user interaction first.

**Comment by minbaowang** [comment]
> Most browsers don’t allow audio to be played automatically, they require user interaction first.
However, when I refresh the page, when the list is loaded, I will automatically pop up the pop-up box in the left column, and JS realizes automatic interaction.

**Comment by minbaowang** [comment]
> > If the wavesurfer is set to play automatically, it will not succeed and an error will be reported；
> 
> Maybe is due to the autoplay policy?

This is a strange one. When I don't remember that the session storage will pop up automatically in the left column, the first time I come in is the list page. Click a line in the list to pop up the left column. At this time, it can be played automatically

**Comment by minbaowang** [comment]
> > The transfer of Vue will be opened automatically.
> 
> Please make an example that reproduces this problem that doesn't use vue.js or any other framework.
##When you click the button, the recording will be generated and played automatically; when the browser has sessionstorage, the recording will be automatically generated instead of clicking the button, but the playback fails,.
[audio_wave.zip](https://github.com/katspaugh/wavesurfer.js/files/5362690/audio_wave.zip)

**Comment by minbaowang** [comment]
> > Vue的传输将自动打开。
> 
> 请举一个重现此问题的示例，该示例不使用vue.js或任何其他框架。



> > The transfer of Vue will be opened automatically.
> 
> Please make an example that reproduces this problem that doesn't use vue.js or any other framework.

I think it's Chrome that restricts automatic playback, but I didn't find a similar implementation of audio's muted attribute on the official website

**Comment by sundayz** [comment]
Chrome and Safari will restrict automatic playback, usually the user has to press a button, and then audio successfully plays. 

Is the DOMException coming from wavesurfer? If you print it with console.error does it give more details?

**Comment by entonbiba** [comment]
@minbaowang browsers will auto play muted sounds by default or if a user has previously clicked the audio player on the same domain.

I've created a pr for a similar issue on mobile devices that are in silent mode that don't play audio since it requires user interaction. On ios browser I created a workaround a few months ago which uses timeouts and once a user touch event happens it will trigger the proper interaction play on a empty audio player allowing the rest of the players to work automatically.
